### ISSUE TITLE
ECS_INSTANCE_ATTRIBUTES not show up

### ISSUE THREAD

**Issue by pilgrim2go** [aws/amazon-ecs-agent#740]
Hi,
I tried to set ECS_INSTANCE_ATTRIBUTES in /etc/ecs/ecs.config  like this

```
ECS_CLUSTER=ECSCluster-6BFMIS0YQP6C
ECS_INSTANCE_ATTRIBUTES={"custom attribute": "custom_attribute_value"}
```
restarted ecs-agent

but I can't find that _custom attribute_ in both Console (View/Edit Attribute) or CLI (_describe-container-instances_)

Is it an issue?

**Comment by pilgrim2go** [comment]
I tried 

```
sudo rm -rf /var/lib/ecs/data/
```

remove ecs-agent

```
docker rm -f ecs-agent
```
and then restart ecs-agent

```
sudo start ecs
```

Doing these above steps, I'm now seeing the custom attribute showing up. But I'm not sure that's the right way to do since I also found there are lots of _ghost_ container instances showing in the UI ( I have to select agent not connected container instance and drained them all).

Will try #713 that is setting up custom attribute at boot time too see.

**Comment by liwenwu-amazon** [comment]
@pilgrim2go Please try #713. We will update README and document to make it clear that
ECS_INSTANCE_ATTRIBUTES only take effect on initial registration.  Once the agent has joined your ECS cluster, you can add additional attributes through the PutAttributes API

**Comment by cabbruzzese** [comment]
The documentation update is being addressed in PR #742

**Comment by adnxn** [comment]
closing issue. readme updated in PR https://github.com/aws/amazon-ecs-agent/pull/742
